packages/utils/scss/layout/_position.scss

```diff
@@ -58,3 +58,153 @@ $kendo-utils-inset: map-get( $kendo-utils, "inset" ) !default;
     left: 50%;
     transform: translateX(-50%);
 }
+
+
+/// @name k-pos-top
+/// @description This is equivalent to `top: 0;`.
+/// @group position
+
+/// @name k-pos-right
+/// @description This is equivalent to `right: 0;`.
+/// @group position
+
+/// @name k-pos-bottom
+/// @description This is equivalent to `bottom: 0;`.
+/// @group position
+
+/// @name k-pos-left
+/// @description This is equivalent to `left: 0;`.
+/// @group position
```

Comment:
no needs of docs for these, as we will deprecate them in favor of the above-generated map at some point after the release, thus please remove these